cmd/kubeapps-apis/plugins/kapp_controller/packages/v1alpha1/server_test.go

```diff
@@ -47,6 +48,53 @@ func TestGetAvailablePackagesStatus(t *testing.T) {
 			},
 			statusCode: codes.FailedPrecondition,
 		},
+		{
+			name: "returns failed-precondition when configGetter itself errors",
+			clientGetter: func(context.Context) (dynamic.Interface, error) {
+				return nil, fmt.Errorf("Bang!")
+			},
+			statusCode: codes.FailedPrecondition,
```

Comment:
Perhaps I'm missing something, but, isn't it the same tests as above? :P

Reply:
Woops, thanks.